client/shipping-label/state/actions.js

```diff
@@ -102,6 +102,10 @@ export const submitStep = ( stepName ) => ( dispatch, getState, { storeOptions }
 	expandFirstErroneousStep( dispatch, getState, storeOptions, stepName );
 };
 
+const convertToApiPackage = ( pckg ) => {
+	return _.pick( pckg, [ 'id', 'box_id', 'service_id', 'length', 'width', 'height', 'weight' ] );
+};
+
```

Comment:
This functionality existed in both PHP (purchasing labels) and JS (requesting rates), so it's basically code deduplication. I kept the JS bit, as we seem to put more logic stuff on the JS side.